mewarpx/tests/test_diagnostic_plotting.py

```diff
@@ -0,0 +1,212 @@
+"""
```

Comment:
Two things:

1. Please name the test `test_field_diag.py` or something like that to show which file it is testing in the `mewarpx` library
2. We should make use of the diode template as much as possible, that way if an API changes in the future we won't have to manually fix several different test input files. So please do make use of the template for this test.